api/src/pcapi/core/providers/constants.py

```diff
@@ -0,0 +1 @@
+CINEMA_PROVIDER_NAMES = ["CDSStocks"]
```

Comment:
```suggestion
CINEMA_PROVIDER_NAMES = [
    "CDSStocks",
]
```
Je suis optimiste, je pars sur l'hypothèse qu'il y en aura vraiment d'autres et cela économisera du _diff_.